cli_tools/gce_vm_image_import/importer/bootable_disk_processor.go

```diff
@@ -128,9 +129,6 @@ func (b bootableDiskProcessor) postValidateFunc() daisy.WorkflowModifier {
 			}}
 		rl.LabelResources(w)
 		daisy_utils.UpdateAllInstanceNoExternalIP(w, b.noExternalIP)
-		if b.uefiCompatible {
```

Comment:
This is deleted because UEFI_COMPATIBLE has been determine in inspection stage and will be inherited. No need to add it again in translation stage.
It's based on an assumption that "--inspect" is enabled automatically. If "--inspect" won't be enabled soon, I will have to add it back. @EricEdens please give some inputs here, thanks!

Reply:
I would keep it and add a TODO bug until it can safely be removed.

Reply:
Sounds good, let me add it back in this PR

Reply:
I went through the scenarios again. Even without "--inspect", this logic can also be removed.
Today, when people enforced "isUefiCompatible" by arg (by API), it's honored in inflation stage, and will be inherited by the final image.

Reply:
sgtm